gradle/buildSrc/src/main/kotlin/io/spine/gradle/internal/deps.kt

```diff
@@ -0,0 +1,303 @@
+    val googleHttpClient       = "com.google.http-client:google-http-client:${Versions.httpClient}"
+    val googleHttpClientApache = "com.google.http-client:google-http-client-apache:${Versions.apacheHttpClient}"
+    val appengineApi           = "com.google.appengine:appengine-api-1.0-sdk:${Versions.appengineApi}"
+    val firebaseAdmin          = "com.google.firebase:firebase-admin:${Versions.firebaseAdmin}"
+    val jacksonDatabind        = "com.fasterxml.jackson.core:jackson-databind:${Versions.jackson}"
+    val roasterApi             = "org.jboss.forge.roaster:roaster-api:${Versions.roaster}"
+    val roasterJdt             = "org.jboss.forge.roaster:roaster-jdt:${Versions.roaster}"
+    val animalSniffer          = "org.codehaus.mojo:animal-sniffer-annotations:${Versions.animalSniffer}"
+    val ci = "true".equals(System.getenv("CI"))
+    val gradlePlugins = GradlePlugins
+}
+
+object Gen {
+    val javaPoet = "com.squareup:javapoet:${Versions.javaPoet}"
+}
+
+object Grpc {
+    val grpcCore               = "io.grpc:grpc-core:${Versions.grpc}"
+    val grpcStub               = "io.grpc:grpc-stub:${Versions.grpc}"
+    val grpcOkHttp             = "io.grpc:grpc-okhttp:${Versions.grpc}"
+    val grpcProtobuf           = "io.grpc:grpc-protobuf:${Versions.grpc}"
+    val grpcNetty              = "io.grpc:grpc-netty:${Versions.grpc}"
+    val grpcNettyShaded        = "io.grpc:grpc-netty-shaded:${Versions.grpc}"
+    val grpcContext            = "io.grpc:grpc-context:${Versions.grpc}"
+}
+ 
+object Runtime {
+    val floggerSystemBackend = "com.google.flogger:flogger-system-backend:${Versions.flogger}"
+    val floggerLog4J         = "com.google.flogger:flogger-log4j:${Versions.flogger}"
+    val floggerSlf4J         = "com.google.flogger:slf4j-backend-factory:${Versions.flogger}"
+}
+
+object Test {
+    val junit4        = "junit:junit:${Versions.junit4}"
+    val junit5Api = listOf(
+            "org.junit.jupiter:junit-jupiter-api:${Versions.junit5}",
+            "org.junit.jupiter:junit-jupiter-params:${Versions.junit5}",
+            "org.apiguardian:apiguardian-api:${Versions.apiguardian}"
+    )
+    val junit5Runner  = "org.junit.jupiter:junit-jupiter-engine:${Versions.junit5}"
+    val junitPioneer  = "org.junit-pioneer:junit-pioneer:${Versions.junitPioneer}"
+    val slf4j         = "org.slf4j:slf4j-jdk14:${Versions.slf4j}"
+    val guavaTestlib  = "com.google.guava:guava-testlib:${Versions.guava}"
+    val mockito       = "org.mockito:mockito-core:2.12.0"
+    val hamcrest      = "org.hamcrest:hamcrest-all:1.3"
+    val truth = listOf(
+            "com.google.truth:truth:${Versions.truth}",
+            "com.google.truth.extensions:truth-java8-extension:${Versions.truth}",
+            "com.google.truth.extensions:truth-proto-extension:${Versions.truth}"
+    )
+}
+
+object Scripts {
+    fun testArtifacts(p: Project)          = "${p.rootDir}/config/gradle/test-artifacts.gradle"
```

Comment:
Maybe we could extract the common path portion into a function? Not sure why exactly, but it's now much more text than previously :)

Reply:
Yes, have a method, please.